src/components/common/YourCompany/CorrectName/CorrectName.vue

```diff
@@ -157,6 +161,9 @@ export default class CorrectName extends Vue {
     if (this.isFormValid) {
       this.isLoading = true
       this.formType = this.currentFormType
+      if (this.currentFormType === 'correct-name-to-number') {
+        this.setNameChangedToNumber(true)
+      }
```

Comment:
Can this be done in the parent instead?

There's a [shared component](https://github.com/bcgov/bcrs-shared-components/tree/main/src/components/correct-name) for the components in this folder but it's slightly incompatible with the implementation here. There is [ticket 20827](https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/20827) to use the shared component here so I don't want to make this code mode incompatible if possible.